### PR TITLE
feat: add power state to VMSS virtual machines

### PR DESCRIPTION
**Pull request by pdecat**
This PR add power state to the `azure_compute_virtual_machine_scale_set_vm` table.

Largely inspired from the code of the `azure_compute_virtual_machine` table.

### CODE REVIEW

**Comment by misraved** [comment]
@pdecat thank you so much for the PR 👍 !!

Could you please add a simple example query showing the value of `power_state` column?

File: azure/table_azure_compute_virtual_machine_scale_set_vm.go [hunk 1]
```diff
@@ -56,6 +56,13 @@ func tableAzureComputeVirtualMachineScaleSetVm(_ context.Context) *plugin.Table
 				Type:        proto.ColumnType_BOOL,
 				Transform:   transform.FromField("VirtualMachineScaleSetVMProperties.LatestModelApplied"),
 			},
+			{
+				Name:        "power_state",
+				Description: "Specifies the power state of the vm.",
```
[the remark this answers is not in the document]
Comment (pdecat) [reply azure/table_azure_compute_virtual_machine_scale_set_vm.go:61]: Copied that from the other table, will fix it in both then :)

[the remark this answers is not in the document]
Comment (pdecat) [reply azure/table_azure_compute_virtual_machine_scale_set_vm.go]: Done.

**Comment by pdecat** [comment]
Added a sample query, here's the result when all instances are in `running` state:
```
> select
  vm.name,
  vm.power_state
from
  azure_compute_virtual_machine_scale_set_vm as vm;
+------------------------------+-------------+
| name                         | power_state |
+------------------------------+-------------+
| aks-frontend-53375067-vmss_0 | running     |
| aks-tooling-45209782-vmss_0  | running     |
| aks-frontend-53375067-vmss_2 | running     |
| aks-default-31213497-vmss_0  | running     |
| aks-service-39716677-vmss_2  | running     |
| aks-service-39716677-vmss_1  | running     |
| aks-frontend-53375067-vmss_1 | running     |
| aks-service-39716677-vmss_3  | running     |
| aks-frontend-53375067-vmss_3 | running     |
| aks-service-39716677-vmss_0  | running     |
+------------------------------+-------------+

```

**Review by misraved — APPROVED** [review]

**Pull request merged**